keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -402,129 +408,124 @@ type AdminNetworkPolicy struct {
+  // ANPStatus is the status to be reported by the implementation, this is not
+  // standardized in alpha and consumers should report what they see fit in
   // relation to their AdminNetworkPolicy implementation
-  // +optional 
+  // +optional
   Status  AdminNetworkPolicyStatus
 }
 
-type AdminNetworkPolicyStatus struct { 
-  conditions   []metav1.Condition  
+type AdminNetworkPolicyStatus struct {
+  Conditions   []metav1.Condition
 }
 
 // AdminNetworkPolicySpec provides the specification of AdminNetworkPolicy
 type AdminNetworkPolicySpec struct {
-  // Priority is an int32 value bound to 0 - 1000, the lowest positive priority, 
-  // "1" corresponds to the lowest importance, while higher priorities have 
-  // higher importance. An ANP with a priority of "0" will be evaluated after all 
+  // Priority is an int32 value bound to 0 - 1000, the lowest positive priority,
+  // "1" corresponds to the lowest importance, while higher priorities have
+  // higher importance. An ANP with a priority of "0" will be evaluated after all
   // positive priority AdminNetworkPolicies and standard NetworkPolicies.
-  // The Priority for an ANP must be set 
+  // The Priority for an ANP must be set
   Priority    *int32
 
   // Subject defines the objects to which this AdminNetworkPolicy applies.  
   Subject     AdminNetworkPolicySubject
 
   // List of Ingress rules to be applied to the selected objects.
-  // A total of 100 rules will be allowed per each network policy instance, 
-  // this rule count will be calculated as the total summation of the 
-  // Ingress and Egress rules in a single AdminNetworkPolicy Instance. 
+  // A total of 100 rules will be allowed per each network policy instance,
+  // this rule count will be calculated as the total summation of the
+  // Ingress and Egress rules in a single AdminNetworkPolicy Instance.
   Ingress     []AdminNetworkPolicyIngressRule
 
   // List of Egress rules to be applied to the selected objects.
-  // A total of 100 rules will be allowed per each network policy instance, 
-  // this rule count will be calculated as the total summation of the 
-  // Ingress and Egress rules in a single AdminNetworkPolicy Instance. 
+  // A total of 100 rules will be allowed per each network policy instance,
+  // this rule count will be calculated as the total summation of the
+  // Ingress and Egress rules in a single AdminNetworkPolicy Instance.
   Egress      []AdminNetworkPolicyEgressRule
 }
 
-// AdminNetworkPolicySubject defines what objects the policy selects. 
-// Exactly one of the selector pointers should be set.  
+// AdminNetworkPolicySubject defines what objects the policy selects.
+// Exactly one of the `Namespaces` or `NamespacedPods` pointers should be set.
 type AdminNetworkPolicySubject struct {
-  NamespaceSelector         *metav1.LabelSelector
-  NamespaceAndPodSelector   *NamespaceAndPodSelector
+  Namespaces        *metav1.LabelSelector
```

Comment:
Based on https://github.com/kubernetes/enhancements/pull/2522#discussion_r794843344 lets not rename this for now since it's a bit confusing that the new *Set Objects have the same yaml name, and this wasn't updated in the sample specs

Reply:
Done